networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -200,18 +226,18 @@ def dfs_postorder_nodes(G,source=None):
     -----
     Based on http://www.ics.uci.edu/~eppstein/PADS/DFS.py
     by D. Eppstein, July 2004.
-
+    https://en.wikipedia.org/wiki/Depth-limited_search
     If a source is not specified then a source is chosen arbitrarily and
     repeatedly until all components in the graph are searched.
     """
-    post=(v for u,v,d in nx.dfs_labeled_edges(G,source=source)
-          if d['dir']=='reverse')
+    post = (v for u, v, d in nx.dfs_labeled_edges(G, source=source,
```

Comment:
Maybe better to do

    edges = nx.dfs_labeled_edges(...)
    return (v for u, v, d in edges if d['dir'] == 'reverse')

to avoid the line breaks altogether.